hacker-news-app/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import { Route, Link } from 'react-router-dom';
+import './App.css';
+import axios from 'axios';
+import LogInForm from './Components/LogInPage/logInPage';
+import CommunityPage from './Components/Community/communityPage.js';
+import Toolbar from './Components/Toolbar/Toolbar';
+import Sidedrawer from './Components/Sidedrawer/Sidedrawer';
+import Backdrop from './Components/Backdrop/Backdrop';
+import Register from './Components/Forms/Register';
+
+function App() {
+  const [users, setUsers] = useState([]);
+  const [error, setError] = useState(null);
+  const [hnData, sethnData] = useState([]);
+  const [drawer,setDrawer] = React.useState(false)
```

Comment:
Lets keep code consistent - `useState` is imported at the top so we don't have to use `React.useState()` anymore

Reply:
Sorry. I didn't noticed that was used. I have sort this out now as well as the irrelevant css styling.